src/fiat_currencies_handling.py

```diff
@@ -0,0 +1,25 @@
+from aiogram.types import (
+    InlineKeyboardButton,
+    InlineKeyboardMarkup,
+    KeyboardButton,
+    ReplyKeyboardMarkup,
+)
+from dataclasses import fields
+
+from env import Env
+from messages import msg
+
+
+async def fiat_handling(message):
+    """
+    fiat currencies handling
+    """
+
+    fiat_menu_buttons = {}
+    for field in fields(msg.fiat_currencies):
+        fiat_menu_buttons[field.name] = KeyboardButton(field.default)
+
+    menu = ReplyKeyboardMarkup(resize_keyboard=True).add(*fiat_menu_buttons.values())
+
+    await message.reply(msg.common_messages.fiat_menu, reply_markup=menu)
+    Env().current_menu[message.chat_id] = "fiat_menu"
```

Comment:
But why do we need to save the state of users' menu? Will fiat menu have something more than "show me current exchange rates"?

Reply:
In this menu there will be a set of the most popular currencies. If the user wants to know other currencies, he will write a message in the {currency_code} format and the bot will have to process this message. Handling such messages from the main menu will not be obvious to users. Similar functionality will be in the cryptocurrencies handling.

Reply:
Oh, if I understand correctly, we can implement that logic by aiogram States:

https://docs.aiogram.dev/en/latest/examples/finite_state_machine_example.html